generators/app/index.js

```diff
@@ -1,10 +1,34 @@
 var generators = require('yeoman-generator');
+var inquirer = require("inquirer");
+var chalk = require('chalk');
+var yosay = require('yosay');
 
 module.exports = generators.Base.extend({
-  method1: function () {
-    console.log('method 1 just ran');
-  },
-  method2: function () {
-    console.log('method 2 just ran');
-  }
+
+	initializing: function() {
+		this.store = {
+			"bower-dependencies": []
+		}
+	},
+
+  	writing: function() {
+  		// App directory structure
+  		this.directory('app', 'app');
+
```

Comment:
Ojo con la indentación/formato del código. No se con que editor lo habéis hecho, pero es posible que este mezcalndo tabuladores con espacios en blanco. Como norma estándar para todos los IDEs, lo que solemos hacer es configurar que el tabulador se convierta en cuatro espacios en blanco.